pkg/operator/ceph/cluster/cephstatus.go

```diff
@@ -257,3 +266,65 @@ func (c *cephStatusChecker) conditionMessageReason(condition cephv1.ConditionTyp
 
 	return condition, reason, message
 }
+
+// forceDeleteStuckPodsOnNotReadyNodes lists all the nodes that are in NotReady state and
+// gets all the pods on the failed node and force delete the pods stuck in terminating state.
+func (c *cephStatusChecker) forceDeleteStuckRookPodsOnNotReadyNodes() error {
+	nodes, err := k8sutil.GetNotReadyKubernetesNodes(c.context.Clientset)
+	if err != nil {
+		return errors.Wrap(err, "failed to get NotReady nodes")
+	}
+	for _, node := range nodes {
+		pods, err := c.getRookPodsOnNode(node.Name)
+		if err != nil {
+			logger.Errorf("failed to get pods on NotReady node %q. %v", node.Name, err)
+		}
+		for _, pod := range pods {
+			if err := k8sutil.ForceDeletePodIfStuck(c.context, pod); err != nil {
+				logger.Errorf("skipping forced delete of stuck pod %q. %v", pod.Name, err)
+			}
+		}
+	}
+	return nil
+}
+
+func (c *cephStatusChecker) getRookPodsOnNode(node string) ([]v1.Pod, error) {
+	clusterName := c.clusterInfo.NamespacedName()
+	labels := []string{
```

Comment:
```suggestion
	appLabels := []string{
```

Makes below code more clear.